src/react-vite/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -191,7 +198,7 @@ Remove these generated files that we won’t need. Some projects do need them, b
 - `src/assets`
 - `src/App.css`
 - `tsconfig.node.json`
-- `.eslintrc.cjs`
+- `.eslintrc.cjs` or `eslint.config.js`
 
 ✏️ Uninstall unneeded packages and Install our eslint config and prettier:
```

Comment:
In the future, we should split this into two commands so it’s more clear and easier to copy/paste.